pkg/ingester/ingester_test.go

```diff
@@ -4460,7 +4464,7 @@ func TestIngester_idleCloseEmptyTSDB(t *testing.T) {
 		return i.lifecycler.HealthyInstancesCount()
 	})
 
-	db, err := i.getOrCreateTSDB(userID, true)
+	db, err := i.getOrCreateTSDB(ctx, userID, true)
```

Comment:
somewhat out of scope, but compactBlocks and shipBlocks further down use context.Backnground(), should we change them to use ctx now?

Reply:
Thanks, done.